index.html

```diff
@@ -33,28 +33,212 @@ <h1 class="headline">
+                            </div>
+                            <div class="tech-item">
+                                css
+                            </div>
+                            <div class="tech-item">
+                                JavScript
+                            </div>
+                            <div class="tech-item">
+                                html
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
+
+                <div class="grid-item">
+                    <div class="grid-item-info">
+                        <h1 class="card-title">Multi-Post Stories Gain+Glory</h1>
+                        <div class="tech-list">
+                            <div class="tech-item">
+                                Ruby on rails
+                            </div>
+                            <div class="tech-item">
+                                css
+                            </div>
+                            <div class="tech-item">
+                                JavScript
+                            </div>
+                            <div class="tech-item">
+                                html
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
 
+                <div class="grid-item">
+                    <div class="grid-item-info">
+                        <h1 class="card-title">Multi-Post Stories Gain+Glory</h1>
+                        <div class="tech-list">
+                            <div class="tech-item">
+                                Ruby on rails
+                            </div>
+                            <div class="tech-item">
+                                css
+                            </div>
+                            <div class="tech-item">
+                                JavScript
+                            </div>
+                            <div class="tech-item">
+                                html
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="center-content">
+            <h1 class="headline about-me">About me</h1>
+            <p class="supporting-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <button type="button" class="see-project-btn">
+                Get My Resume
+            </button>
```

Comment:
Please add the background that exists in the Figma design
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/153575899-74989dbf-e856-4071-a520-96351a840376.png) | ![image](https://user-images.githubusercontent.com/24703171/153576030-30e6dc75-352c-47d6-93d2-a9602c6b64de.png) |